flake8_quotes/__init__.py

```diff
@@ -30,13 +30,15 @@ def __init__(self, tree, filename='(none)', builtins=None):
 
     @classmethod
     def add_options(cls, parser):
-        parser.add_option('--quotes', default='\'', action='store',
+        parser.add_option('--quotes', default='\'', action='store', dest='inline_quotes',
```

Comment:
Yep, but if we have `--inline-quotes` as the primary option and someone was using the default already, then there should be no issue with there being no default set on `--quotes`.